frontend/index.html

```diff
@@ -1 +1,24 @@
-<!-- O FRONTEND SERÁ DESENVOLVIDO DENTRO DESTE DIRETORIO -->
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pokédex</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 class="pkHeader">Pokédex</h1>
+    <div class="caixa">
+        <label for="pkInput">Digite o nome ou número do Pokémon:</label>
```

Comment:
Seria interessante alterar para "Nome ou Numero da Pokedex, pois fica mais subentendido qual numero se refere, devido aos jogos de pokemon terem inúmeras numerações diferentes.